### PR TITLE
KUZBO-137 Index summary

### PR DESCRIPTION
**Pull request by jenow**
Provides the summary page for a given index.

### CODE REVIEW

**Comment by stafyniaksacha** [comment]
+1

**Comment by jenow** [comment]
+1

File: src/components/Data/Browse/IndexBranch.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div :class="{ 'open': open }">
     <i v-if="collectionCount(index)" class="fa fa-caret-right tree-toggle" aria-hidden="true" @click="toggleBranch()"></i>
     <!-- v-link="{name: 'DataIndex', params: {index: index.name}}" -->
```
Comment (xbill82) [inline src/components/Data/Browse/IndexBranch.vue]: Let's take this comment out :)

File: src/vuex/modules/data/store.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
-  RECEIVE_MAPPING
+  RECEIVE_MAPPING,
+  RECEIVE_COLLECTIONS
 } from './mutation-types'
 
 const state = {
   indexesAndCollections: undefined,
-  mapping: undefined
+  mapping: undefined,
+  collections: {
```
Comment (xbill82) [inline src/vuex/modules/data/store.js]: **Not to be changed in this PR, but...**
I'm afraid that this naming is too "broad" and that we'll soon be in trouble understanding what `collections` are we talking about... We have the collections that are shown in the Tree View (which are organized by index), then we have this one `collection`, which contains the ones that are shown in the Index Summary page (so they're basically "The List Of Collections Of the Currently Browsed Index", let's find a better name for that).
Then, do we have any other collection list? May we take five minutes to think about it together?

**Comment by xbill82** [comment]
:+1:

**Pull request merged**